packages/myst-common/src/extractParts.ts

```diff
@@ -38,6 +54,7 @@ export function extractPart(tree: GenericParent, part: string): GenericParent |
         delete block.data.tags;
       }
     }
+    if (opts?.removePartData) delete block.data.part;
```

Comment:
This doesn't delete the part data if it's in `tags` - is that an edge case we should keep consistent?

Reply:
Hm, yeah, this needs to be fixed. Will add a test. Thanks.